cli.go

```diff
@@ -43,26 +43,35 @@ func New(opts map[string]interface{}) *Cli {
 	endpoint, _ := opts["endpoint"].(string)
 	url, _ := url.Parse(strings.TrimRight(endpoint, "/"))
 
-	transport := &http.Transport{
-		TLSClientConfig: &tls.Config{},
-	}
-
 	if project, ok := opts["project"].(string); ok {
 		opts["project"] = strings.ToUpper(project)
 	}
 
-	if insecureSkipVerify, ok := opts["insecure"].(bool); ok {
-		transport.TLSClientConfig.InsecureSkipVerify = insecureSkipVerify
+	var ua *http.Client
+	if unixProxyPath, ok := opts["unixproxy"].(string); ok {
```

Comment:
the main/main.go will need to be modified to allow for this option I think (look for `optigo.NewDirectAssignParser`), and the usage should be updated to list the option and specify format.  (I was not sure if this should be just a path, or if it should have a `unix://` prefix ... maybe should allow for both?)
-Cory